index.js

```diff
@@ -1,3 +1,16 @@
+import { createCharacterCard } from "./components/card/card.js";
+
+/* fetch('https://rickandmortyapi.com/api/character').then(response => { 
+  if (response.ok) { 
+    return response.json(); 
+  }
+  throw new Error("request failed") 
+
+}, networkerror => console.log(networkerror.message).then(jsonResponse => { 
+
+})
+);  */
```

Comment:
It's okay for now, but you shouldn't leave code in comments. If there's code you need, use it. Otherwise, remove it.